### PR TITLE
CASMTRIAGE-6844 1.3 : DOCS: cray-console-data-postgres has leader but start failed

### PR DESCRIPTION
**Pull request by kimjensen-hpe**
# Description
Resolve CASMTRIAGE-6844 for release/1.3

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/kubernetes/Troubleshoot_Postgres_Database.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ This page contains general Postgres troubleshooting topics.
     - [Case 1: `some persistent volumes are not compatible with existing resizing providers`](#case-1-some-persistent-volumes-are-not-compatible-with-existing-resizing-providers)
     - [Case 2: `could not init db connection`](#case-2-could-not-init-db-connection)
     - [Case 3: `password authentication failed for user`](#case-3-password-authentication-failed-for-user)
+- [Cluster member start failed](#cluster-member-start-failed)
+  - [Determine if invalid permissions](#determine-if-invalid-permissions)
+  - [Recover start failed member](#recover-from-a-start-failed-member)
```
Comment (bo-quan) [inline operations/kubernetes/Troubleshoot_Postgres_Database.md]:
```suggestion
  - [Recover from a start failed member](#recover-from-a-start-failed-member)
```

File: operations/kubernetes/Troubleshoot_Postgres_Database.md [hunk 2]
```diff
@@ -873,6 +876,68 @@ Kubernetes secret and update the password in the database.
...
+kubectl logs "${POSTGRESQL}-0" -n ${NAMESPACE} -c postgres | grep FATAL
+```
+
+Example output:
+
+```text
+2024-03-30 07:41:35 UTC [2092847]: [1-1] 6607c22f.1fef2f 0     FATAL:  data directory "/home/postgres/pgdata/pgroot/data" has invalid permissions
+```
+
+### Recover from a start failed
```
Comment (bo-quan) [inline operations/kubernetes/Troubleshoot_Postgres_Database.md]:
```suggestion
### Recover from a start failed member
```

**Review by bo-quan — CHANGES_REQUESTED** [review]

**Review by bo-quan — APPROVED** [review]

File: operations/kubernetes/Troubleshoot_Postgres_Database.md (hunk 1 above)
Comment (kimjensen-hpe) [reply operations/kubernetes/Troubleshoot_Postgres_Database.md]: fixed

File: operations/kubernetes/Troubleshoot_Postgres_Database.md (hunk 2 above)
Comment (kimjensen-hpe) [reply operations/kubernetes/Troubleshoot_Postgres_Database.md]: fixed

**Review by rustydb — APPROVED** [review]

**Pull request merged**